modules/gui/view_logs.py

```diff
@@ -0,0 +1,94 @@
+from modules.app.helper import Vector2
+
+from modules.gui.gui_module import GuiModule
+
+from tkinter import *
+from tkinter.font import BOLD
+from tkinter.scrolledtext import ScrolledText
+
+from modules.logging import Logging
+
+class GUI_ViewLogs( GuiModule ):
+
+    def drawLog( self, frame ):
+        header_frame = Frame(frame)
+        header_frame.pack(side=TOP, fill='x', pady=0, padx=(0,20))
+
+        table = Frame(header_frame)
+        table.pack(fill=BOTH, expand=1)
+
+        # Create the columns
+        ip_label = Label(table, text="IP", width=15)
+        ip_label.grid(row=0, column=0)
+        date_label = Label(table, text="Date", width=15)
+        date_label.grid(row=0, column=1)
+        content_label = Label(table, text="Content", width=30)
+        content_label.grid(row=0, column=2)
+
+        log = self.context.read_write.getLogFile()
+        for i, entry in enumerate(log):
+            ip_entry = Entry(table, width=15)
+            ip_entry.grid(row=i+1, column=0)
+            ip_entry.insert(0, f"{entry['ip']}")
```

Comment:
Perhaps an option could be added to display the hostname in addition to (or instead of) the IP address? (Retrieving it with https://github.com/mhoek2/tcp-share/blob/30b9e0eb779f3c7955fab94dd74e6de93107f821/modules/app/settings.py#L35.)
Absolutely not necessary, of course, but it could be nice to have (and more descriptive).

Reply:
Not a bad idea

Reply:
I changed the layout of tables to display the device ip and hostname on top of the table
![table](https://github.com/user-attachments/assets/a7832b05-f09e-4709-9df7-a4dde5e9fde3)

Reply:
Nice, looks good, and it works on my school PC.  
The program crashes when starting it on a computer which doesn’t have an IP address that’s listed in `LAN_devices`, though (at least, I presume that is the reason); perhaps some kind of fallback can be added for cases like that (having the hostname then be something like `n/a`).

Reply:
Crash sounds valid, can implement a fallback indeed.

Reply:
f1b175a98b09c14f70a9deb82e5ef5264ce3a7b4
This will add a ```n/a``` for the hostname but still displays the IP address